packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -111,7 +110,7 @@ export class WorkspaceMainImpl implements WorkspaceMain, Disposable {
     }
 
     $pickWorkspaceFolder(options: WorkspaceFolderPickOptionsMain): Promise<theia.WorkspaceFolder | undefined> {
-        return new Promise((resolve, reject) => {
+        return new Promise(async (resolve, reject) => {
```

Comment:
why `async`? i don't see you added `await` anywhere

Reply:
Fixed.